.github/workflows/requirements.yml

```diff
@@ -0,0 +1,66 @@
+# cspell:ignore noreply
+
+name: Upgrade requirements
+
+on:
+  schedule:
+    - cron: "0 2 * * 1"
+
+jobs:
+  requirements:
+    name: Upgrade requirement files
```

Comment:
This only checks out the default branch right?

Reply:
Seems like it, yes

> The branch, tag or SHA to checkout. When checking out the repository that triggered a workflow, this defaults to the reference or SHA for that event. _Otherwise, uses the default branch._

https://github.com/actions/checkout#usage